charts/openebs/templates/ndm/daemonset-ndm.yaml

```diff
@@ -63,6 +63,9 @@ spec:
 {{- if .Values.featureGates.ChangeDetection.enabled }}
           - --feature-gates={{ .Values.featureGates.ChangeDetection.featureGateFlag }}
 {{- end}}
+{{- if .Values.featureGates.PartitionTableUUID.enabled }}
```

Comment:
q: This file is used only in pre 3.0 releases right? After that we use the dependent charts. In that case, should we add new modifications to this file? or anything in the templates directory?

Reply:
ChangeDetection is there.
Also, this NDM is used for dynamic-localpv-provisioner (device)

Also, for the dynamic-localpv-provisioner, both legacy and the subchart are updated

Reply:
Also from a usability perspective, it might be difficult for a user to have two very similar NDMs, one with new features, and one without.